src/wat-writer.cc

```diff
@@ -1380,7 +1380,13 @@ void WatWriter::WriteTable(const Table& table) {
 
 void WatWriter::WriteElemSegment(const ElemSegment& segment) {
   WriteOpenSpace("elem");
-  WriteNameOrIndex(segment.name, elem_segment_index_, NextChar::Space);
+  // The name there pre-bulk-memory was intended to refer to the table
```

Comment:
typo here? I can't really parse this sentence, s/there/that?

Reply:
I re-write this .. let me know if it make more sense now.

Reply:
Thanks!